Language/Variables/Data Types/float.adoc

```diff
@@ -16,24 +16,24 @@ subCategories: [ "Data Types" ]
 --
 
 [float]
-=== Description
-Datatype for floating-point numbers, a number that has a decimal point. Floating-point numbers are often used to approximate analog and continuous values because they have greater resolution than integers. Floating-point numbers can be as large as 3.4028235E+38 and as low as -3.4028235E+38. They are stored as 32 bits (4 bytes) of information.
+=== Descrizione
+Tipo di dato per i numeri in virgola mobile, un numero che ha un punto decimale. I numeri in virgola mobile sono spesso utilizzati per approssimare valori analogici e continui, perché hanno una risoluzione maggiore rispetto ai numeri interi. I numeri in virgola mobile possono essere grandi fino a 3,4028235E+38 e minimi fino a -3,4028235E+38. Vengono memorizzati in 32 bit (4 byte) di informazioni.
```

Comment:
```suggestion
Tipo di dato per i numeri in virgola mobile, un numero che ha un punto decimale. I numeri in virgola mobile sono spesso utilizzati per approssimare valori analogici e continui, perché hanno una precisione maggiore rispetto ai numeri interi. I numeri in virgola mobile possono essere grandi fino a 3,4028235E+38 e piccoli fino a -3,4028235E+38. Vengono memorizzati in 32 bit (4 byte) di informazioni.
```

Reply:
:-) Sounds good this kind of revisions @g-castronovo